src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -5351,7 +5352,7 @@ void Assembler::vpmovzxbw(XMMRegister dst, Address src, int vector_len) {
   InstructionMark im(this);
   assert(dst != xnoreg, "sanity");
   InstructionAttr attributes(vector_len, /* rex_w */ false, /* legacy_mode */ _legacy_mode_bw, /* no_mask_reg */ true, /* uses_vl */ true);
-  attributes.set_address_attributes(/* tuple_type */ EVEX_HVM, /* input_size_in_bits */ EVEX_NObit);
+  attributes.set_address_attributes(/* tuple_type */ EVEX_QVM, /* input_size_in_bits */ EVEX_NObit);
```

Comment:
This should have remained as EVEX_HVM.

Reply:
Correct.
https://www.felixcloutier.com/x86/pmovzx#:~:text=VPMOVZXBW%20xmm1%20%7Bk1%7D%7Bz%7D%2C%20xmm2/m64-,B,-V/V
https://www.felixcloutier.com/x86/pmovzx#:~:text=N/A-,B,Half%20Mem,-ModRM%3Areg%20(w

Good.